meetalva/alva
# feat: associate .alva files on macOS

## Todo

- [x] Associate .alva files
- [x] Replace .icns with better small icons
- [x] Check if Alva opens with files
- [x] Open file also during Alva startup

## Steps to test
* npm run build:electron
* install .dmg
* double-click on .alva file

Should be squased into `feat: associate .alva files on macOS`
